docker/Dockerfile

```diff
@@ -106,6 +112,9 @@ WORKDIR $CONTAINER_SOURCE/
 # Upstream only
 COPY --from=cleanup --chown=1001:1001 $CONTAINER_SOURCE/ ./
 
+COPY --chown=1001:1001 docker/install-dynamic-plugins.py ./
+COPY --from=dynamic-plugins --chown=1001:1001 $CONTAINER_SOURCE/ ./
```

Comment:
should this be copying only the `$CONTAINER_SOURCE/dynamic-plugins/` folder?

Reply:
True.
I'll rework this part anyway.

Reply:
looks OK